libnetwork/iptables/iptables_test.go

```diff
@@ -196,6 +210,21 @@ func TestConcurrencyNoWait(t *testing.T) {
 // Note that if iptables does not support the xtable lock on this
 // system, then allowXlock has no effect -- it will always be off.
 func RunConcurrencyTest(t *testing.T, allowXlock bool) {
+	iptable, natChain, filterChain, err := createNewChain()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = iptable.ProgramChain(natChain, bridgeName, false, true)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = iptable.ProgramChain(filterChain, bridgeName, false, true)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
How come the chains have to be programmed a second time? Why aren't the `ProgramChain` calls inside `createNewChain` sufficient?

Reply:
Hi @corhere , thanks for reviewing! 

Thanks for catching this, I can't see any reasons why they are needed, it should be my mistake. I have removed them.